deltachat-ffi/deltachat.h

```diff
@@ -5033,10 +5033,18 @@ int             dc_contact_is_blocked        (const dc_contact_t* contact);
 
 
 /**
- * Check if a contact was verified. E.g. by a secure-join QR code scan
- * and if the key has not changed since this verification.
+ * Check if the contact
+ * can be added to verified chats,
+ * i.e. has a verified key
+ * and Autocrypt key matches the verified key.
  *
- * The UI may draw a checkbox or something like that beside verified contacts.
+ * If contact is verified
+ * UI should display green checkmark after the contact name
+ * in the title of the contact profile,
+ * in contact list items and in chat member list items.
+ *
+ * Do not use this function when displaying profile view contents.
```

Comment:
```suggestion
 * Do not use this function when displaying profile view "Introduced by" contents.
```

i would be more precise here, as in the profile's title bar, using is_verified() is fine (to be consistent with lists)